CHANGELOG.md

```diff
@@ -4,6 +4,160 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.7.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.7.0) - 2019-01-01
+* BREAKING
+  * Refactor heatmap to vue component (#5401)
```

Comment:
I'd put this with the original heatmap pr in the list (line 19), as the heatmap didn't exist in previous releases.